src/compiler/filesystem_local_win.cc

```diff
@@ -42,18 +42,27 @@ bool FilesystemLocal::is_absolute(const char* path) {
   if (SourceManager::is_virtual_file(path)) return true;
   int length = strlen(path);
   if (length < 3) return false;
-  return path[1] == ':' && path[2] == PATH_SEPARATOR;
+  // Either "c:\" or "\\wsl$"
```

Comment:
You don't check for the `wsl$` part so the comment is a bit confusing for me.

Reply:
Added better comments.